app/models/exporters/csv_exporter.rb

```diff
@@ -44,7 +44,12 @@ def headers(fields)
 
   def row(record, fields)
     [record.id] + fields.map do |field|
-      record.data[field.name]
```

Comment:
Why do you need this change?

Reply:
Values in csv were not populating for all the subform fields.
![Screenshot 2022-03-09 16:41:45](https://user-images.githubusercontent.com/5393463/157435174-c37f447f-ecea-4d3e-9b84-675de617b832.png)